src/resources/License/LicenseInterfaces.ts

```diff
@@ -0,0 +1,27 @@
+export interface LicenseConnectorScheduleModel {
+    refreshType: string;
+    scheduleFrequencies: string[];
+}
+
+export interface LicenseConnectorModel {
+    allowed: boolean;
+    allowedSchedules: LicenseConnectorScheduleModel[];
+    sourceVisibilities: string[];
+    type: string;
+}
+
+export interface LicenseModel {
+    accountId: string;
+    accountName: string;
+    connectors: LicenseConnectorModel[];
+    department: string;
+    indexBackupType: string;
+    indexType: string;
+    monitoringLevel: string;
+    productEdition: string;
+    productName: string;
+    productType: string;
+    properties: any;
+    expirationDate: number;
+    type: string;
+}
```

Comment:
Some fields should probably be optional in those. I think we could add someone from the backend team to validate.

Reply:
That's the backend response, not the data that I send